website/docs/plugins/ray_launcher.md

```diff
@@ -134,6 +134,80 @@ multirun
                     └── checkpoint_3.pt
 ```
 
+Same as the `ray_local` launcher, you can discover the `ray_aws` launcher's config with:
+
+```commandline
```

Comment:
This is big enough that I it makes sense to fold the yaml output with a detail tag.